client/src/Questionnaires/Questionnaires.js

```diff
@@ -18,6 +19,9 @@ function Questionnaires() {
       <AuthProtectedRoute path={`${path}/:id/edit`}>
         <QuestionnaireForm />
       </AuthProtectedRoute>
+      <Route path={`${path}/:id/questions`}>
+        <Questions />
+      </Route>
```

Comment:
Magic! See how we can re-use the EXACT SAME Questions component that sets up the Questions paths, and make it a NESTED sub-route of Questionnaires... in this way, we'll be able to look at/filter just the questions for a given questionnaire, whose id is in the URL...